### ISSUE TITLE
Modify layout for empty events band

### ISSUE THREAD

**Issue by skoolbus39** [unl/wdntemplates#1314]
When no upcoming events are present in the events band either display a message that there are currently no events or not display the band at all.

![Screen Shot 2019-05-08 at 9 33 59 AM](https://user-images.githubusercontent.com/5385125/57383898-3bce5100-7175-11e9-8491-5e8fd6e4d8ac.png)

**Comment by Dharmesh-Poddar** [comment]
hii @skoolbus39 in events section when I search for a day having no event dated I get  "0 search results for events dated". Can you please elaborate the issue

**Comment by skoolbus39** [comment]
Hi @Dharmesh-Poddar, are you expecting a different result if you search for a day that has no events?

**Comment by Dharmesh-Poddar** [comment]
Yes @skoolbus39 I think I am getting different result from what you specified
